user-guide/Standard_Apps/Monitoring_Solutions/DataMiner_Objects_Tool/Using_DataMiner_Objects_Tool.md

```diff
@@ -0,0 +1,74 @@
+
+For **reservation instances** (2), the app shows the total number of reservation instances in the system along with a pie chart that groups the reservation instances by state. The button next to these stats launches the same *DataMiner Objects Tool* script, but immediately opens the dialog to filter reservation instances.
+
+For **resources** (3), the app shows the total number of resources in the system along with a pie chart that groups the resources by resource pool. The button next to these stats launches the same *DataMiner Objects Tool* script, but immediately opens the dialog to filter resources.
+
+For **DOM instances** (4), the app shows the total number of DOM instances in the system along with a pie chart that groups the DOM instances by module. The button next to these stats launches the same *DataMiner Objects Tool* script, but immediately opens the dialog to filter DOM instances.
+
+## Configuring filters
+
+To configure a filter:
+
+![Configuring filters](~/user-guide/images/Using_DataMiner_Objects_Tool_ConfiguringFilters.png)
+
+1. **Include or exclude filters** by selecting or clearing the checkbox next to an item property name.
+
+1. Select the **filter type** in the dropdown boxes. Different properties allow different filter types depending on their possible values.
+
+1. Fill in the **filter value**. Depending on the property you filter on, this can be a string, integer, GUID, or datetime.
+
+1. If you want to **add more filters**, use the *Duplicate* button.
+
+   There is no limit to the number of filters you can include.
+
+1. Click the button at the bottom of the window to **execute the query** with the included filters.
+
+> [!IMPORTANT]
+> All included filters will be combined with an AND operation. There is currently no way of adding OR filters.
+
+## Fine-tuning results
+
+Once the items have been retrieved based on the filters you have configured, you can manually fine-tune the selection:
+
+![Fine-tuning results](~/user-guide/images/Using_DataMiner_Objects_Tool_FinetuningResults.png)
+
+- The **number of matching items** (1) is displayed to give a quick view on how many items in the system match the filters.
+
+- Buttons are available to **select all or unselect all items** (2).
+
+- The **number of selected items** (3) is also displayed, so you can see how many are included in your current selection.
+
+- In the scrollable checkbox list at the bottom (4), you can **select individual items** or clear the selection to fine-tune the result according to your preferences.
+
+## Executing actions on selected objects
+
+After you have manually fine-tuned the results, you can execute the following actions for the selected objects:
+
+![Actions](~/user-guide/images/Using_DataMiner_Objects_Tool_Actions.png)
+
+- To add a new multi-line textbox to the dialog containing all selected objects serialized in JSON format, click **Show JSON** (1).
```

Comment:
What would this JSON format typically be used for? It might be useful if we added this here, so users searching for this will come across this specific page.

Reply:
The JSON is used to see which fields of the object have which value and is mainly used to investigate/debug issues. Can also be used to export the object and import it into any other application. Should I add these 2 main uses for the JSON?

Reply:
Looking good! Thanks!